### PR TITLE
Fix sharing's methods

### PR DESCRIPTION
**Pull request by RafaRuiz**
Fix sharing's methods needed to share locks

### CODE REVIEW

File: common/services/deviceOperation.js [hunk 1]
```diff
@@ -64,6 +64,16 @@ const _executor = function (baseUrl, deviceId, operation) {
...
+      'Authorization': 'Bearer ' + localStorage.token,
+      'Content-Type': 'application/json'
+    }
+  }).then(response => {
+    return response
+  })
+}
 const _getUserByEmail = function (baseUrl, userEmail) {
   return axios
       .post(baseUrl + "/share/invite/" + userEmail, null, {
```
Comment (mpbarnwell) [inline common/services/deviceOperation.js:79]: Shouldn't you remove the `_getUserByEmail` call if you are adding `_getDoordeckUserByEmail`?

**Review by mpbarnwell — CHANGES_REQUESTED** [review]
Don't

File: common/services/deviceOperation.js (hunk 1 above)
Comment (RafaRuiz) [reply common/services/deviceOperation.js:79]: they have different usages:

cloud uses `getUser`:

image

and somehow this is the real method out of it:

image

Comment (mpbarnwell) [reply common/services/deviceOperation.js:79]: It's the same endpoint, all you've done is add the visitor flag to one

Comment (RafaRuiz) [reply common/services/deviceOperation.js:79]: I'll add a flag then

**Review by mpbarnwell — APPROVED** [review]

**Pull request merged**